core/document/src/document.rs

```diff
@@ -227,6 +227,11 @@ impl Document {
 		Ok(())
 	}
 
+	pub fn layer_bounding_box(&self, path: &[LayerId]) -> Result<[DVec2; 2], DocumentError> {
```

Comment:
Please rename `layer_axis_aligned_bounding_box` and create another function called `layer_local_bounding_box` which returns the bounding box of the layer's own rotation.

Reply:
Done